src/components/list/list.scss

```diff
@@ -138,11 +138,13 @@ md-list-item {
       align-items: center;
       justify-content: flex-start;
 
-      padding: 0;
+      padding: 0 16px;
       margin: 0;
 
+      // The button should not inherit the parents background color.
+      background-color: initial;
```

Comment:
Already talked about that with Thomas, IE11 doesn't need this property, because the bug isn't present there. So this will be just an unnecessary property on IE11